types/server.js

```
@@ -410,123 +569,185 @@ class FabricHTTPServer extends Service {
+                method: 'JSONCallResult',
+                params: [hash, result]
+              })]).signWithKey(this._rootKey);
+
+              socket.send(callResultMessage.toBuffer());
+            } catch (exception) {
+              console.error('[SERVER]', 'Could not parse JSON blob:', exception);
+              return;
+            }
+            break;
+          case 'GET':
+            let answer = await server._GET(message['@data']['path']);
```

Biome (2.4.9)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)